pkg/cfg/config.go

```diff
@@ -177,8 +177,8 @@ func (c *config) GetMsiSlice(key string, optionalDefault ...[]map[string]interfa
 	var reflectValue = reflect.ValueOf(data)
 
 	if reflectValue.Kind() != reflect.Slice {
-		err = fmt.Errorf("can not cast value %v[%T] of key %s to []map[string]interface{}", data, data, key)
-		c.err(err, err.Error())
+		err = fmt.Errorf("can not cast value %v[%T] of key %s to []map[string]interface{}: %w", data, data, key, err)
+		c.err(err.Error())
```

Comment:
Shouldn't this be `c.err("can not cast value %v[%T] of key %s to []map[string]interface{}: %w", data, data, key, err)`?

Reply:
indeed